build.gradle.kts

```diff
@@ -140,6 +142,31 @@ subprojects {
 		}
 
 		afterEvaluate {
+			if (project != project(":junit-jupiter")) {
+				tasks["jar"].withConvention(aQute.bnd.gradle.BundleTaskConvention::class) {
+					setBnd("""
+						Bundle-Name: ${project.description}
+						Import-Package: \
+							!org.apiguardian.api,\
+							kotlin.*;resolution:="optional",\
+							*
+						-fixupmessages.kotlin.import: "Unused Import-Package instructions: \\[kotlin.*\\]";is:=ignore
+						-removeheaders: Private-Package
+						-noextraheaders: true
+
+						# The following exports only packages having classes annotated @API
+						# First collect all classes annotated with @API
+						possibleAPI: ${'$'}{classes;HIERARCHY_INDIRECTLY_ANNOTATED;org.apiguardian.api.API}
```

Comment:
Cool. I didn't know you could do things like that with `bnd`. 👍 

But... that will automatically export `org.junit.jupiter.engine.config` (which should not be exported because all classes annotated with `@API` in that package are actually `@API(status = INTERNAL, ...)`) -- right?

So it looks like we will still have to fine tune things a bit.

Or can you do even more magic with `bnd` to check the `status` attribute of `@API` declarations?